pkgs/pkgs-lib/formats/hocon/test/comprehensive/default.nix

```diff
@@ -37,7 +37,7 @@ let
         "b"
       ]
     ];
-    nasty_string = "\"@\n\\\t^*\b\f\n\0\";'''$";
+    nasty_string = "\"@\n\\\t^*bf\n0\";'''$";
```

Comment:
Nasty indeed.

Reply:
I think it would be more interesting if that string actually contained the characters it intended to contain (except the NUL byte, because that horribly breaks Nix …), but I couldn't figure out how to easily put a line feed into a string without actually putting verbatim into the file, so …